src/gui/systray.mm

```diff
@@ -58,7 +58,7 @@ void sendTalkReply(UNNotificationResponse *response, UNNotificationContent* cont
                             << "Token:" << qToken
                             << "Account:" << qAccount;
 
-    QPointer<OCC::TalkReply> talkReply = new OCC::TalkReply(accountState.data(), OCC::Systray::instance());
+    const auto talkReply = QSharedPointer<OCC::TalkReply>(new OCC::TalkReply(accountState.data(), OCC::Systray::instance()));
```

Comment:
@claucambra A minor thing, but since you're here, you might as well wanna use `QSharedPointer<T>::create` instead.

Reply:
Thanks, changed this